packages/react-magma-dom/src/components/AlertBase/index.tsx

```diff
@@ -211,12 +255,14 @@ const IconWrapper = styled.span<{ isToast?: boolean; theme: any }>`
     props.theme.spaceScale.spacing04};
 
   @media (max-width: ${props => props.theme.breakpoints.small}px) {
-    display: none;
+    padding: 0 8px;
```

Comment:
Should we use the `spacing03` variable here?

Reply:
Seems reasonable to me.